zomes/wiki/code/src/page.rs

```diff
@@ -95,75 +84,95 @@ pub fn page_def() -> ValidatingEntryType {
 
     )
 }
-pub fn create_page_if_non_existent(title: String) -> ZomeApiResult<Address> {
-    let address = Page::initial(title.clone()).entry().address();
-    match hdk::get_entry(&address)? {
-        None => {
-            let page_anchor = utils::anchor("wiki_pages", "all_pages")?;
-            hdk::utils::commit_and_link(
-                &Page::initial(title.clone()).entry(),
-                &page_anchor,
-                "anchor->Page",
-                &title,
-            )
-        }
-        Some(_) => Ok(address),
+pub fn create_page_if_non_existent(
+    title: String,
+    sections: Vec<Address>,
+    timestamp: String,
+) -> ZomeApiResult<Address> {
+    let anchor_address = holochain_anchors::anchor("wiki_pages".to_string(), title.clone())?;
+    if let Some(address) = hdk::get_links(
+        &anchor_address,
+        LinkMatch::Exactly("anchor->Page"),
```

Comment:
Can you change all instances of `anchor->Page` to `anchor->page`? For consistency

Reply:
create_page_if_not_existent does not make sense if it receives a list of sections: if it receives the sections but is already created it will not do anything. It's better to create a `create_page_anchor` without any section input and that only creates the anchor when it does not exist, and then a `create_page_entry(title: String, page: Page)` that calls create_page_anchor, creates the entry and calls `link_anchor_to_page`.